tests/api/test_generic.py

```diff
@@ -50,32 +50,3 @@ async def test_get_extensions_no_user(client):
     response = await client.get("extensions")
     # bad request
     assert response.status_code == 401, f"{response.url} {response.status_code}"
-
-
-# check GET /extensions: enable extension
-# TODO: test fails because of removing lnurlp extension
-# @pytest.mark.asyncio
-# async def test_get_extensions_enable(client, to_user):
-#     response = await client.get(
-#         "extensions", params={"usr": to_user.id, "enable": "lnurlp"}
-#     )
-#     assert response.status_code == 200, f"{response.url} {response.status_code}"
-
-
-# check GET /extensions: enable and disable extensions, expect code 400 bad request
-# @pytest.mark.asyncio
-# async def test_get_extensions_enable_and_disable(client, to_user):
-#     response = await client.get(
-#         "extensions",
-#         params={"usr": to_user.id, "enable": "lnurlp", "disable": "lnurlp"},
-#     )
-#     assert response.status_code == 400, f"{response.url} {response.status_code}"
-
-
-# check GET /extensions: enable nonexistent extension, expect code 400 bad request
-@pytest.mark.asyncio
-async def test_get_extensions_enable_nonexistent_extension(client, to_user):
-    response = await client.get(
-        "extensions", params={"usr": to_user.id, "enable": "12341234"}
-    )
-    assert response.status_code == 400, f"{response.url} {response.status_code}"
```

Comment:
- this logic has been moved out of `/extensions` and into `/api/v1/extension`
- it is now covered by JMeter integration tests